### PR TITLE
feat(ld-notification): use up-to-date css custom props

### PR DESCRIPTION
**Pull request by borisdiakur**
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/liquid/components/ld-notification/ld-notification.css [hunk 1]
```diff
@@ -1,4 +1,19 @@
 :host {
+  /* layout */
+  --ld-notification-min-height: var(--ld-sp-40);
+  --ld-notification-border-radius: var(--ld-br-m);
+  --ld-notification-inset-y: var(--ld-sp-32);
+
+  /* colors */
+  --ld-notification-col: var(--ld-col-wht);
+  --ld-notification-warn-col: var(--ld-col-neutral);
+  --ld-notification-alert-col: var(--ld-col-wht);
```
Comment (renet) [inline src/liquid/components/ld-notification/ld-notification.css:10]:
```suggestion
  --ld-notification-text-col: var(--ld-col-wht);
  --ld-notification-warn-text-col: var(--ld-col-neutral);
  --ld-notification-alert-text-col: var(--ld-col-wht);
```

File: src/liquid/components/ld-notification/ld-notification.css [hunk 2]
```diff
@@ -114,16 +129,16 @@
 }
 
 .ld-notification__item--info {
-  background-color: var(--ld-notification-item-color);
-  color: var(--ld-col-wht);
+  background-color: var(--ld-notification-bg-col);
+  color: var(--ld-notification-col);
```
Comment (renet) [inline src/liquid/components/ld-notification/ld-notification.css:133]:
```suggestion
  color: var(--ld-notification-text-col);
```

File: src/liquid/components/ld-notification/ld-notification.css [hunk 3]
```diff
@@ -114,16 +129,16 @@
...
 }
 .ld-notification__item--warn {
-  background-color: var(--ld-col-vy-default);
-  color: var(--ld-col-rblck-default);
+  background-color: var(--ld-notification-warn-bg-col);
+  color: var(--ld-notification-warn-col);
```
Comment (renet) [inline src/liquid/components/ld-notification/ld-notification.css:137]:
```suggestion
  color: var(--ld-notification-warn-text-col);
```

File: src/liquid/components/ld-notification/ld-notification.css [hunk 4]
```diff
@@ -114,16 +129,16 @@
...
 }
 .ld-notification__item--alert {
-  background-color: var(--ld-col-rr-default);
-  color: var(--ld-col-wht);
+  background-color: var(--ld-notification-alert-bg-col);
+  color: var(--ld-notification-alert-col);
```
Comment (renet) [inline src/liquid/components/ld-notification/ld-notification.css:141]:
```suggestion
  color: var(--ld-notification-alert-text-col);
```

File: screenshot/builds/master.json [hunk 5]
```diff
@@ -17035,4 +17035,4 @@
       "isMobile": false
     }
   ]
-}
+}
```
Comment (renet) [inline screenshot/builds/master.json:17038]: We keep pushing this newline back and forth. Might this be related to the cleanup script?

**Review by renet — CHANGES_REQUESTED** [review]

File: src/liquid/components/ld-notification/ld-notification.css (hunk 1 above)
Comment (borisdiakur) [reply src/liquid/components/ld-notification/ld-notification.css:10]: I'll rather keep it as it is. In css it's either background-color or color. There is no text-color. So I'll stick to this.

File: src/liquid/components/ld-notification/ld-notification.css (hunk 2 above)
Comment (borisdiakur) [reply src/liquid/components/ld-notification/ld-notification.css:133]: Same here.

File: src/liquid/components/ld-notification/ld-notification.css (hunk 3 above)
Comment (borisdiakur) [reply src/liquid/components/ld-notification/ld-notification.css:137]: And here.

File: src/liquid/components/ld-notification/ld-notification.css (hunk 4 above)
Comment (borisdiakur) [reply src/liquid/components/ld-notification/ld-notification.css:141]: "

**Review by renet — APPROVED** [review]

**Pull request merged**